Description: [pkg-config] Add libgpg-error to gnutls' Libs.private.
Author: Andreas Metzler <ametzler@debian.org>
Bug-Debian: http://bugs.debian.org/632891
Forwarded: no (Upstream has switched to nettle as crypto backend)

--- gnutls26-2.12.7.orig/lib/gnutls.pc.in
+++ gnutls26-2.12.7/lib/gnutls.pc.in
@@ -20,6 +20,6 @@ Description: Transport Security Layer im
 URL: http://www.gnu.org/software/gnutls/
 Version: @VERSION@
 Libs: -L${libdir} -lgnutls
-Libs.private: @LTLIBGCRYPT@ @NETTLE_LIBS@
+Libs.private: @LTLIBGCRYPT@ @NETTLE_LIBS@ -lgpg-error
 @GNUTLS_REQUIRES_PRIVATE@
 Cflags: -I${includedir}
